src/renderer.ts

```diff
@@ -251,51 +250,6 @@ const text = (doc: DocumentFragment, pillar: Pillar): ReactNode[] =>
 const standfirstText = (doc: DocumentFragment, pillar: Pillar): ReactNode[] =>
     Array.from(doc.childNodes).map(standfirstTextElement(pillar));
 
-interface ImageProps {
-    url: string;
-    alt: string;
-    imageMappings: ImageMappings;
-    sizes: string;
-    width: number;
-    height: number;
-    captionString: string;
-    caption: DocumentFragment;
-    credit: string;
-    pillar: Pillar;
-}
-
-const imageStyles = (width: number, height: number): SerializedStyles => css`
-    height: calc(100vw * ${height / width});
-    background: ${neutral[97]};
```

Comment:
Should we keep this? we have it in the templates to show where an image will load in

Reply:
Yes good spot, looks like I lost that somewhere moving code around.